src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1497,7 +1497,7 @@ If such a class is used as a base class and if the deriving class defines a dest
     <value>'{0}' in explicit interface declaration is not an interface</value>
   </data>
   <data name="ERR_InterfaceMemberNotFound" xml:space="preserve">
-    <value>'{0}' in explicit interface declaration is not a member of interface</value>
+    <value>'{0}' in explicit interface declaration is not found among members of the interface that can be implemented</value>
   </data>
```

Comment:
I agree, given that you now use the same diagnostic for other situations, it needs to be more vague to cover those additional situations. My point is the the different causes of failure should have distinct diagnostics. Thank you for opening the tracking item.